<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>DialogMatrix</class>
  <widget class="QDialog" name="DialogMatrix">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>462</width>
        <height>253</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Matrix Dialog</string>
    </property>
    <layout class="QGridLayout">
      <property name="margin">
        <number>0</number>
      </property>
      <property name="spacing">
        <number>0</number>
      </property>
      <item row="0" column="0">
        <layout class="QVBoxLayout">
          <property name="margin">
            <number>1</number>
          </property>
          <property name="spacing">
            <number>1</number>
          </property>
          <item>
            <layout class="QHBoxLayout">
              <item>
                <widget class="QLabel" name="textLabel1_2">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>3</hsizetype>
                      <vsizetype>5</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;Zoom&lt;/p&gt;</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer1">
                  <property name="sizeHint">
                    <size>
                      <width>31</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QLineEdit" name="m_pEditZoom">
                  <property name="minimumSize">
                    <size>
                      <width>45</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="maximumSize">
                    <size>
                      <width>45</width>
                      <height>32767</height>
                    </size>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QSlider" name="m_pSliderZoom">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QHBoxLayout">
              <item>
                <widget class="QLabel" name="textLabel1">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>3</hsizetype>
                      <vsizetype>5</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;Rotate&lt;/p&gt;</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer2">
                  <property name="sizeHint">
                    <size>
                      <width>30</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QLineEdit" name="m_pEditRotate">
                  <property name="minimumSize">
                    <size>
                      <width>45</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="maximumSize">
                    <size>
                      <width>45</width>
                      <height>32767</height>
                    </size>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QSlider" name="m_pSliderRotate">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QHBoxLayout">
              <item>
                <widget class="QLabel" name="textLabel1_2_2">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>7</hsizetype>
                      <vsizetype>5</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;Shear X&lt;/p&gt;</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer3">
                  <property name="sizeHint">
                    <size>
                      <width>31</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QLineEdit" name="m_pEditShearX">
                  <property name="minimumSize">
                    <size>
                      <width>45</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="maximumSize">
                    <size>
                      <width>45</width>
                      <height>32767</height>
                    </size>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QSlider" name="m_pSliderShearX">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QHBoxLayout">
              <item>
                <widget class="QLabel" name="textLabel1_3">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>3</hsizetype>
                      <vsizetype>5</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>&lt;p align=&quot;center&quot;&gt;Shear Y&lt;/p&gt;</string>
                  </property>
                  <property name="wordWrap">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer4">
                  <property name="sizeHint">
                    <size>
                      <width>30</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </spacer>
              </item>
              <item>
                <widget class="QLineEdit" name="m_pEditShearY">
                  <property name="minimumSize">
                    <size>
                      <width>45</width>
                      <height>0</height>
                    </size>
                  </property>
                  <property name="maximumSize">
                    <size>
                      <width>45</width>
                      <height>32767</height>
                    </size>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QSlider" name="m_pSliderShearY">
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="m_pButtonClose">
              <property name="text">
                <string>Close</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="global">qmatrix.h</include>
    <include location="local">menuobject.h</include>
  </includes>
  <connections>
    <connection>
      <sender>m_pButtonClose</sender>
      <signal>clicked()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotClose()</slot>
    </connection>
    <connection>
      <sender>m_pSliderZoom</sender>
      <signal>valueChanged(int)</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotZoom(int)</slot>
    </connection>
    <connection>
      <sender>m_pSliderRotate</sender>
      <signal>valueChanged(int)</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotRotate(int)</slot>
    </connection>
    <connection>
      <sender>m_pSliderShearX</sender>
      <signal>valueChanged(int)</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotShearX(int)</slot>
    </connection>
    <connection>
      <sender>m_pSliderShearY</sender>
      <signal>valueChanged(int)</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotShearY(int)</slot>
    </connection>
    <connection>
      <sender>m_pSliderZoom</sender>
      <signal>sliderReleased()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderReleased()</slot>
    </connection>
    <connection>
      <sender>m_pSliderRotate</sender>
      <signal>sliderReleased()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderReleased()</slot>
    </connection>
    <connection>
      <sender>m_pSliderShearX</sender>
      <signal>sliderReleased()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderReleased()</slot>
    </connection>
    <connection>
      <sender>m_pSliderShearY</sender>
      <signal>sliderReleased()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderReleased()</slot>
    </connection>
    <connection>
      <sender>m_pSliderRotate</sender>
      <signal>sliderPressed()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderPressed()</slot>
    </connection>
    <connection>
      <sender>m_pSliderShearX</sender>
      <signal>sliderPressed()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderPressed()</slot>
    </connection>
    <connection>
      <sender>m_pSliderShearY</sender>
      <signal>sliderPressed()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderPressed()</slot>
    </connection>
    <connection>
      <sender>m_pSliderZoom</sender>
      <signal>sliderPressed()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotSliderPressed()</slot>
    </connection>
    <connection>
      <sender>m_pEditZoom</sender>
      <signal>lostFocus()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotZoomChanged()</slot>
    </connection>
    <connection>
      <sender>m_pEditRotate</sender>
      <signal>lostFocus()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotRotateChanged()</slot>
    </connection>
    <connection>
      <sender>m_pEditShearX</sender>
      <signal>lostFocus()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotShearXChanged()</slot>
    </connection>
    <connection>
      <sender>m_pEditShearY</sender>
      <signal>lostFocus()</signal>
      <receiver>DialogMatrix</receiver>
      <slot>slotShearYChanged()</slot>
    </connection>
  </connections>
</ui>
